### PR TITLE
[luci-interpreter] Test quantized DepthwiseConv2D kernel for MobilenetV2

### PR DESCRIPTION
**Pull request by struss**
This commit update `DepthwiseConv2D` operator test case.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
OK to merge?

File: compiler/luci-interpreter/src/kernels/DepthwiseConv2D.test.cpp [hunk 1]
```diff
@@ -68,6 +68,67 @@ TEST(DepthwiseConv2DTest, Float)
   };
   EXPECT_THAT(extractTensorData<float>(output_tensor),
               ElementsAreArray(ArrayFloatNear(ref_output_data)));
+  EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray({1, 2, 1, 4}));
+}
+
+TEST(DepthwiseConv2DTest, Uint8)
+{
+  float kQuantizeTolerance = getTolerance(-127, 128, 255);
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/DepthwiseConv2D.test.cpp:76]: Please be consistent with naming. We usually use snake_case for variable names.

Comment (struss) [reply compiler/luci-interpreter/src/kernels/DepthwiseConv2D.test.cpp:76]: Oh, Sorry for make mistake. I will fix it.

Comment (struss) [reply compiler/luci-interpreter/src/kernels/DepthwiseConv2D.test.cpp]: this variable is not used, so I remove it.

**Pull request merged**